### DISCUSSION TITLE
diy hot key

### DISCUSSION THREAD

**Discussion by snowxuyu** [dbeaver/dbeaver#14449] [Q&A] [answered]
I hope to customize the shortcut, such as the speed of executing the SQL statement in the Mac system is Command + ENTER, not Control + Enter.

**Comment by andrelinslima** [comment, marked as answer]
Shortcuts are customizable using the builtin eclipse functionality:
Window -> Preference -> User Interface -> Keys
You can search for Execute SQL Statement for example and change the keys for that

**Comment by snowxuyu** [comment]
I found it. Thank you！
